.github/workflows/acceptance-tests.yaml

```diff
@@ -40,45 +40,45 @@ jobs:
     environment:
       name: renku-ci-gw-${{ github.event.number }}
     steps:
-    - name: deploy-pr
-      uses: SwissDataScienceCenter/renku/actions/deploy-renku@master
-      env:
-        DOCKER_PASSWORD: ${{ secrets.RENKU_DOCKER_PASSWORD }}
-        DOCKER_USERNAME: ${{ secrets.RENKU_DOCKER_USERNAME }}
-        GITLAB_TOKEN: ${{ secrets.DEV_GITLAB_TOKEN }}
-        KUBECONFIG: "${{ github.workspace }}/renkubot-kube.config"
-        RANCHER_PROJECT_ID: ${{ secrets.CI_RANCHER_PROJECT }}
-        RENKU_RELEASE: renku-ci-gw-${{ github.event.number }}
-        RENKU_VALUES_FILE: "${{ github.workspace }}/values.yaml"
-        RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
-        RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_DEV_KUBECONFIG }}
-        RENKUBOT_RANCHER_BEARER_TOKEN: ${{ secrets.RENKUBOT_RANCHER_BEARER_TOKEN }}
-        RANCHER_DEV_API_ENDPOINT: ${{ secrets.RANCHER_DEV_API_ENDPOINT }}
-        RENKU_BOT_DEV_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
-        RENKU_ANONYMOUS_SESSIONS: true
-        RENKU_TESTS_ENABLED: true
-        TEST_ARTIFACTS_PATH: "tests-artifacts-${{ github.sha }}"
-        renku_gateway: "@${{ github.head_ref }}"
-        renku: "${{ needs.check-deploy.outputs.renku }}"
-        renku_core: "${{ needs.check-deploy.outputs.renku-core }}"
-        renku_graph: "${{ needs.check-deploy.outputs.renku-graph }}"
-        renku_notebooks: "${{ needs.check-deploy.outputs.renku-notebooks }}"
-        renku_ui: "${{ needs.check-deploy.outputs.renku-ui }}"
-    - name: Check existing renkubot comment
-      uses: peter-evans/find-comment@v1
-      id: findcomment
-      with:
-        issue-number: ${{ github.event.pull_request.number }}
-        comment-author: 'RenkuBot'
-        body-includes: "You can access the deployment of this PR at"
-    - name: Create comment pre deploy
-      if: steps.findcomment.outputs.comment-id == 0
-      uses: peter-evans/create-or-update-comment@v1
-      with:
-        token: ${{ secrets.RENKUBOT_GITHUB_TOKEN }}
-        issue-number: ${{ github.event.pull_request.number }}
-        body: |
-          You can access the deployment of this PR at https://renku-ci-gw-${{ github.event.number }}.dev.renku.ch
+      - name: deploy-pr
+        uses: SwissDataScienceCenter/renku/actions/deploy-renku@master
```

Comment:
don't you mean `SwissDataScienceCenter/renku-actions/deploy-renku@master`?

Reply:
yeah good catch - I made a bit of a mess trying to resolve a simple merge conflict.